src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1016,4 +1016,10 @@ I agree to all of the foregoing:</value>
   <data name="Code_style_header_use_editor_config" xml:space="preserve">
     <value>The settings configured here only apply to your machine. To configure these settings to travel with your solution, use .editorconfig files.</value>
   </data>
-</root>
+  <data name="Analyzing_0" xml:space="preserve">
+    <value>Analyzing '{0}'</value>
+  </data>
+  <data name="Analyzing_errors" xml:space="preserve">
+    <value>Analyzing errors</value>
```

Comment:
💭 To me, this sounds like you have a list of errors already and it's looking for solutions to them.

@dotnet/roslyn-ide What do you think of 'Live code analysis' here?

Reply:
sure.